StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1501UnitTests.cs

```diff
@@ -507,12 +507,12 @@ void MethodName()
     {
         do
         {
-            Bar();
+            MethodName();
         }
         while (false);
 ";
 
-            await this.VerifyCSharpFixAsync(testCode, fixedTestCode).ConfigureAwait(false);
+            await this.VerifyCSharpFixAsync(testCode, fixedTestCode, allowUnsupportedDiagnostics: true).ConfigureAwait(false);
```

Comment:
:question: Why did this one need to be a special case? It seems like a bunch of special behavior was added to the testing framework just to support this single test.